README.md

```diff
@@ -15,6 +15,7 @@ At the moment only bitcoin is supported.
 
 * [Golang](https://golang.org/)
 * [Vault](https://www.vaultproject.io/)
+* [Homebrew (Mac OS X)](https://brew.sh/?utm_source=devmag.io)
```

Comment:
If you put here as a requirement it seems to refer to MacOSX only.

Actually I'm not sure that to use golang is needed if we find the way to ship the plugin as binary

Reply:
it referes to macos only. The fact is that Golang can be installed from `brew` or from `.pkg`

Reply:
I mean it's not a requirement anyway, it's a specific tool needed if you have macosx as OS. Put below with the Mac OSx path

As user we need golang because we need to build the plugin right? If we find a way to release as binary theoretically we don't need it in the machine for Production

Reply:
yes, go needed only for development purpose.